SPECK/src/main/java/SPEck.java

```diff
@@ -227,19 +296,22 @@ protected static void writeDataset(String fileOut, Utils.Dataset dataset) throws
     /**
      * Compute the p-values of the FSP of a dataset merging the p-values computed by the different cores.
```

Comment:
Compute the *non-normalized* p-values (i.e., the number of the T datasets with results as or more extreme than the ones in `fileFSP`).

We can remove the part about "merging ..."